examples/Client/PublishSubscribe/BulkPublishEventExample/BulkPublishEventExample.cs

```diff
@@ -0,0 +1,54 @@
+// ------------------------------------------------------------------------
+// Copyright 2021 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Dapr.Client;
+
+namespace Samples.Client
+{
+    public class BulkPublishEventExample : Example
```

Comment:
Thanks for making an example! Do you think we can have it hook up to the existing example though? Or does the subscribe also need to be different? Cause it'd be cool to see the bank example do multiple deposits.

Reply:
Done